packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -664,6 +684,12 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
     private renderInfoBarConfiguration(): JSX.Element {
         return (
             <div className={this.props.managedClasses.site_infoBarConfiguration}>
+                <button
+                    onClick={this.handleTransparencyToggleClick}
+                    className={this.props.managedClasses.site_transparencyToggleButton}
+                >
+                    <span dangerouslySetInnerHTML={{__html: glyphTransparency}}/>
```

Comment:
I think we can just use the base64 string instead of having a glyph

Reply:
Discussed offline we want to keep these separate for consistency